File: 3731-find-missing-elements/3731-find-missing-elements.java
class Solution {
    public List<Integer> findMissingElements(int[] nums) {
        ArrayList<Integer> list=new ArrayList<>();
        Arrays.sort(nums);
        int start=nums[0];
        int ind=0;
        for(int i=start;i<nums[nums.length-1];i++)
            {
             if(nums[ind]==i)
             {
                 ind++;
             }
            else{
                list.add(i);
            }
            }
        return list;
    }
}

File: 3731-find-missing-elements/3731-find-missing-elements.cpp
class Solution {
public:
    vector<int> findMissingElements(vector<int>& nums) {
        int n=nums.size();
        // int mini=*min_elemeny(nums.begi)
        sort(nums.begin(),nums.end());
        int i=nums[0];
        vector<int> res;
        int ind=0;
        while(i<nums[n-1])
        {
            if(i==nums[ind])
            {
                i++;
                ind++;
            }
            else{
                res.push_back(i);
                i++;
            }
        }
        return res;
    }
};